keyboards/keebfront/vanguard65/info.json

```diff
@@ -0,0 +1,130 @@
+                { "label": "K09", "matrix": [0, 9], "w": 1, "x": 9, "y": 0 },
+                { "label": "K0A", "matrix": [0, 10], "w": 1, "x": 10, "y": 0 },
+                { "label": "K0B", "matrix": [0, 11], "w": 1, "x": 11, "y": 0 },
+                { "label": "K0C", "matrix": [0, 12], "w": 1, "x": 12, "y": 0 },
+                { "label": "K0D", "matrix": [0, 13], "w": 1, "x": 13, "y": 0 },
+                { "label": "K0E", "matrix": [0, 14], "w": 1, "x": 14, "y": 0 },
+                { "label": "K10", "matrix": [1, 0], "w": 1, "x": 15, "y": 0 },
+                { "label": "K11", "matrix": [1, 1], "w": 1.5, "x": 0, "y": 1 },
+                { "label": "K12", "matrix": [1, 2], "w": 1, "x": 1.5, "y": 1 },
+                { "label": "K13", "matrix": [1, 3], "w": 1, "x": 2.5, "y": 1 },
+                { "label": "K14", "matrix": [1, 4], "w": 1, "x": 3.5, "y": 1 },
+                { "label": "K15", "matrix": [1, 5], "w": 1, "x": 4.5, "y": 1 },
+                { "label": "K16", "matrix": [1, 6], "w": 1, "x": 5.5, "y": 1 },
+                { "label": "K17", "matrix": [1, 7], "w": 1, "x": 6.5, "y": 1 },
+                { "label": "K18", "matrix": [1, 8], "w": 1, "x": 7.5, "y": 1 },
+                { "label": "K19", "matrix": [1, 9], "w": 1, "x": 8.5, "y": 1 },
+                { "label": "K1A", "matrix": [1, 10], "w": 1, "x": 9.5, "y": 1 },
+                { "label": "K1B", "matrix": [1, 11], "w": 1, "x": 10.5, "y": 1 },
+                { "label": "K1C", "matrix": [1, 12], "w": 1, "x": 11.5, "y": 1 },
+                { "label": "K1D", "matrix": [1, 13], "w": 1, "x": 12.5, "y": 1 },
+                { "label": "K1E", "matrix": [1, 14], "w": 1.5, "x": 13.5, "y": 1 },
+                { "label": "K20", "matrix": [2, 0], "w": 1, "x": 15, "y": 1 },
+                { "label": "K21", "matrix": [2, 1], "w": 1.75, "x": 0, "y": 2 },
+                { "label": "K22", "matrix": [2, 2], "w": 1, "x": 1.75, "y": 2 },
+                { "label": "K23", "matrix": [2, 3], "w": 1, "x": 2.75, "y": 2 },
+                { "label": "K24", "matrix": [2, 4], "w": 1, "x": 3.75, "y": 2 },
+                { "label": "K25", "matrix": [2, 5], "w": 1, "x": 4.75, "y": 2 },
+                { "label": "K26", "matrix": [2, 6], "w": 1, "x": 5.75, "y": 2 },
+                { "label": "K27", "matrix": [2, 7], "w": 1, "x": 6.75, "y": 2 },
+                { "label": "K28", "matrix": [2, 8], "w": 1, "x": 7.75, "y": 2 },
+                { "label": "K29", "matrix": [2, 9], "w": 1, "x": 8.75, "y": 2 },
+                { "label": "K2A", "matrix": [2, 10], "w": 1, "x": 9.75, "y": 2 },
+                { "label": "K2B", "matrix": [2, 11], "w": 1, "x": 10.75, "y": 2 },
+                { "label": "K2C", "matrix": [2, 12], "w": 1, "x": 11.75, "y": 2 },
+                { "label": "K2D", "matrix": [2, 13], "w": 2.25, "x": 12.75, "y": 2 },
+                { "label": "K2E", "matrix": [2, 14], "w": 1, "x": 15, "y": 2 },
```

Comment:
moves the upper right 3 key cluster outwards a little, to match the physical PCB/board layout
```suggestion
                { "label": "K10", "matrix": [1, 0], "w": 1, "x": 15.25, "y": 0 },
                { "label": "K11", "matrix": [1, 1], "w": 1.5, "x": 0, "y": 1 },
                { "label": "K12", "matrix": [1, 2], "w": 1, "x": 1.5, "y": 1 },
                { "label": "K13", "matrix": [1, 3], "w": 1, "x": 2.5, "y": 1 },
                { "label": "K14", "matrix": [1, 4], "w": 1, "x": 3.5, "y": 1 },
                { "label": "K15", "matrix": [1, 5], "w": 1, "x": 4.5, "y": 1 },
                { "label": "K16", "matrix": [1, 6], "w": 1, "x": 5.5, "y": 1 },
                { "label": "K17", "matrix": [1, 7], "w": 1, "x": 6.5, "y": 1 },
                { "label": "K18", "matrix": [1, 8], "w": 1, "x": 7.5, "y": 1 },
                { "label": "K19", "matrix": [1, 9], "w": 1, "x": 8.5, "y": 1 },
                { "label": "K1A", "matrix": [1, 10], "w": 1, "x": 9.5, "y": 1 },
                { "label": "K1B", "matrix": [1, 11], "w": 1, "x": 10.5, "y": 1 },
                { "label": "K1C", "matrix": [1, 12], "w": 1, "x": 11.5, "y": 1 },
                { "label": "K1D", "matrix": [1, 13], "w": 1, "x": 12.5, "y": 1 },
                { "label": "K1E", "matrix": [1, 14], "w": 1.5, "x": 13.5, "y": 1 },
                { "label": "K20", "matrix": [2, 0], "w": 1, "x": 15.25, "y": 1 },
                { "label": "K21", "matrix": [2, 1], "w": 1.75, "x": 0, "y": 2 },
                { "label": "K22", "matrix": [2, 2], "w": 1, "x": 1.75, "y": 2 },
                { "label": "K23", "matrix": [2, 3], "w": 1, "x": 2.75, "y": 2 },
                { "label": "K24", "matrix": [2, 4], "w": 1, "x": 3.75, "y": 2 },
                { "label": "K25", "matrix": [2, 5], "w": 1, "x": 4.75, "y": 2 },
                { "label": "K26", "matrix": [2, 6], "w": 1, "x": 5.75, "y": 2 },
                { "label": "K27", "matrix": [2, 7], "w": 1, "x": 6.75, "y": 2 },
                { "label": "K28", "matrix": [2, 8], "w": 1, "x": 7.75, "y": 2 },
                { "label": "K29", "matrix": [2, 9], "w": 1, "x": 8.75, "y": 2 },
                { "label": "K2A", "matrix": [2, 10], "w": 1, "x": 9.75, "y": 2 },
                { "label": "K2B", "matrix": [2, 11], "w": 1, "x": 10.75, "y": 2 },
                { "label": "K2C", "matrix": [2, 12], "w": 1, "x": 11.75, "y": 2 },
                { "label": "K2D", "matrix": [2, 13], "w": 2.25, "x": 12.75, "y": 2 },
                { "label": "K2E", "matrix": [2, 14], "w": 1, "x": 15.25, "y": 2 },
```